tests/utils/managed_process.py

```
@@ -549,6 +579,45 @@ def _terminate_process_group(self, timeout: float = 8.0):
+            self.terminate_parent_only_first
+            and self.proc is not None
+            and self.proc.poll() is None
+        ):
+            try:
+                self._logger.info(
+                    "Sending SIGTERM to parent process: %s", self.proc.pid
+                )
+                self.proc.terminate()
+            except ProcessLookupError:
+                pass
+            except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)